zmsadmin/.gitlab-ci.yml

```
@@ -3,8 +3,8 @@ include:
 
 test-php-73:
     variables:
-      PHP_VERSION: 7.3
-    image: registry.gitlab.com/eappointment/php-base:${PHP_VERSION}-dev
+      PHP_VERSION: 8.0
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)